lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -8,6 +8,9 @@
 from galaxy.datatypes.metadata import MetadataElement
 from galaxy import util
 
+from stl import mesh
+import numpy as np
+
```

Comment:
Can you invert this 2 imports and move them before the `galaxy` imports? I.e.:
```
import numpy as np
from stl import mesh

from galaxy.datatypes import data
...
```

We are now following the `smarkets` import style, as in this [example](https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_smarkets.py).